components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -212,17 +220,17 @@ const ContributorHighlightCard = ({
   const handleUpdateHighlight = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
 
-    const pullrequestLink = highlight.prLink.trim();
+    const pullrequestLink = highlight.highlightLink.trim();
     if (wordCount > wordLimit) {
       setError("Character limit exceeded");
       return;
     }
 
     if (isValidPullRequestUrl(pullrequestLink) || isValidIssueUrl(pullrequestLink)) {
-      const { apiPaths } = generateRepoParts(highlight.prLink);
+      const { apiPaths } = generateApiPrUrl(highlight.highlightLink);
```

Comment:
`generateApiPrUrl` is failing the build.

https://app.netlify.com/sites/oss-insights/deploys/64dffbc0ff828f0008b5db74#L2531